detox/e2e/helper/screen.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import env from './env';
+
+/**
+ * fulfillSelectServerScreen enters site URL on Select Server screen and then tap connect button.
+ * @param {String} url - site URL with default supplied from env variable
+ */
+export async function fulfillSelectServerScreen(url = env.siteUrl) {
+    await expect(element(by.id('select_server_screen'))).toBeVisible();
+
+    await element(by.id('server_url_input')).clearText();
+    await element(by.id('server_url_input')).typeText(url);
+
+    // # Tap anywhere to hide keyboard
+    await element(by.text('Enter Server URL')).tap();
+
+    await element(by.id('connect_button')).tap();
```

Comment:
this looks really simple.. can we try and add a test for a more complex scenario?